lib/osf-components/addon/components/maintenance-banner/template.hbs

```diff
@@ -2,6 +2,7 @@
     {{#bs-alert
         type=alertType
         onDismiss=(action 'click' 'button' 'Maintenance Banner - dismiss' target=analytics)
+        onDismiss=(action 'dismiss')
```

Comment:
I think the second `onDismiss` will overwrite the first. Maybe move the analytics into the action?